### ISSUE TITLE
Does it support llama3?

### ISSUE THREAD

**Issue by orderer0001** [OptimalScale/LMFlow#785]
Does it support multi-gpu raining llama3 with lisa ?

**Comment by research4pan** [comment]
Thanks for your interest in LMFlow! We currently support single-gpu training of LLaMA-3-7b with LISA. We are working on the multi-gpu model-parallelism version, hopefully in 1-2 weeks, please stay tuned. Thanks for your understanding 😄

**Comment by orderer0001** [comment]
> LLaMA-3-7b

Do you support llama 2-7b? Or llama 3-8B? llama3-7b does not exist.

**Comment by research4pan** [comment]
Yes we support both models as long as the access is granted from huggingface repo for your huggingface account.
* https://huggingface.co/meta-llama/Llama-2-7b
* https://huggingface.co/meta-llama/Meta-Llama-3-8B

**Comment by orderer0001** [comment]
> Yes we support both models as long as the access is granted from huggingface repo for your huggingface account.
> 
> * https://huggingface.co/meta-llama/Llama-2-7b
> * https://huggingface.co/meta-llama/Meta-Llama-3-8B

Which parameter is used to configure authorized account related information?

**Comment by wheresmyhair** [comment]
> > Yes we support both models as long as the access is granted from huggingface repo for your huggingface account.
> > 
> > * [huggingface.co/meta-llama/Llama-2-7b](https://huggingface.co/meta-llama/Llama-2-7b)
> > * [huggingface.co/meta-llama/Meta-Llama-3-8B](https://huggingface.co/meta-llama/Meta-Llama-3-8B)
> 
> Which parameter is used to configure authorized account related information?

You may first switch to lmflow conda env using
```bash
conda activate YOUR_ENV_NAME
```
then use
```bash
huggingface-cli login
```
and follow the instructions.
